packages/core-snapshots/src/db/queries/rounds/round-range.sql

```diff
@@ -1,12 +1,12 @@
 SELECT
-  id,
-  public_key,
+  round,
   balance,
-  round
+  public_key
 FROM
   rounds
 WHERE
   round BETWEEN ${startRound} AND ${endRound} AND
-  id >= ${startId}
```

Comment:
The query is broken now. Did you test if the snapshotter still works?

Reply:
Fixed in 632425bbd. No, I did not test (obviously).